src/reducers/books.js

```diff
@@ -0,0 +1,41 @@
+import { REMOVE_BOOK, CREATE_BOOK } from '../actions/index';
+
+const books = [
+  {
+    ID: 1,
+    title: 'Title 1',
+    category: 'Cat-1',
+
+  },
+  {
+    ID: 2,
+    title: 'Title 1',
+    category: 'Cat-1',
+
+  },
+  {
+    ID: 3,
+    title: 'Title 1',
+    category: 'Cat-1',
+
+  },
+];
+
+const bookReducer = (state = books, action) => {
+  switch (action.type) {
+    case CREATE_BOOK:
+      return [
+        ...state, action.book,
+      ];
+    case REMOVE_BOOK:
+      return state.filter(book => book.ID !== action.ID);
+      // return [
+      //   ...state.slice(0, action.ID), ...state.slice((action.ID + 1))
+      // ];
```

Comment:
- [ ] Also remove this commented block of code.

Reply:
Done.